distributor-node/package.json

```diff
@@ -13,7 +13,7 @@
     "@joystream/metadata-protobuf": "^2.8.0",
     "@joystream/opentelemetry": "1.0.0",
     "@joystream/storage-node-client": "^3.0.0",
-    "@joystream/types": "^3.0.0",
+    "@joystream/types": "^2.0.0",
```

Comment:
Not clear why this is being reverted .. but I can imagine it was quite difficult to keep things in sync with the multitude of times you rebased

Reply:
It was due to the `yarn build:packages` failing

Reply:
Setting it to ^4.0.0 fixes the CI

Reply:
here https://github.com/Joystream/joystream/pull/4871/commits/ead0209600d711681888c6aff1ed222e07422b4e